src/Test/Utilities/Portable/Compilation/TestOperationVisitor.cs

```diff
@@ -255,6 +260,16 @@ public override void VisitForToLoop(IForToLoopOperation operation)
         {
             VisitLoop(operation);
             Assert.Equal(LoopKind.ForTo, operation.LoopKind);
+            _ = operation.IsChecked;
```

Comment:
This line looks suspicious. It implies there is a property here which is side effecting that needs to be executed.

Reply:
> This line looks suspicious. It implies there is a property here which is side effecting that needs to be executed. 

The purpose of this line is to evaluate the property to make sure it doesn't crash. This is a test code.

---
In reply to: [187121790](https://github.com/dotnet/roslyn/pull/26734#discussion_r187121790) [](ancestors = 187121790)